zt_frontend/src/components/AddCell.vue

```diff
@@ -1,11 +1,13 @@
 <template>
```

Comment:
Few comments here: If I have focus on a cell we should show the add cell component above and below for that cell. Also when a notebook has 0 cells, we should show the add cell bar at the top of the notebook